include/stdexec/stop_token.hpp

```diff
@@ -26,9 +26,9 @@
 #include <thread>
 #include <concepts>
 
-#if __has_include(<stop_token>) && __cpp_lib_jthread >= 201911
-#  include <stop_token>
-#endif
+//#if __has_include(<stop_token>) && __cpp_lib_jthread >= 201911
+//#  include <stop_token>
+//#endif
```

Comment:
why did this get commented out?

Reply:
No longer needed with https://github.com/dvyukov/relacy/pull/23

Relacy replaces standard includes at compile time, but `stop_token` is new compared to relacy.